resume/src/components/sections/Basics.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import Title from "../Title";
+import Info from "../Info";
+import Profiles from "./Profiles";
+
+import "./Sections.css";
+
+export default function Section(props) {
+  let counter = 0;
```

Comment:
alternatively, the callback function passed to ```map``` has a second optional parameter for the current index, so instead of storing your own counter you can do:

```
props.data.profiles.map((profile, counter) => {
    return (
      <Profiles
        network={profile.network}
        username={profile.username}
        url={profile.url}
        key={counter}
      />
    );
```

(UNTESTED CODE)